lib/scarpe/wv/font.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "scarpe/base64"
+
+class Scarpe
+  class WebviewFont < WebviewWidget
+    include Base64
+    attr_accessor :font
+
+    def initialize(properties)
+      @font = properties[:font]
+      super
+    end
+
+    def element
+      puts @font
+      HTML.render do |h|
+        h.link(href: @font, rel: "stylesheet")
+        h.style do
+          <<~CSS
+            @font-face {
+              font-family: Pacifico;
```

Comment:
Huh. Can we change this so it's not always Pacifico?

Reply:
ah oh we can its just a custom name but  like get name from filepath?

Reply:
That sounds like a good way to do it, yeah.

I mean, if we can't customise then we can't, but it seems worth a try.